packages/core/admin/admin/src/features/Notifications.tsx

```diff
@@ -91,7 +91,7 @@ const NotificationsProvider = ({ children }: NotificationsProviderProps) => {
         gap={2}
         top={`${46 / 16}rem`}
         width={`${500 / 16}rem`}
-        zIndex={10}
+        zIndex={999999}
```

Comment:
😬

Reply:
I'm not a fan of this as I feel like we could find items that need a higher z-index. All the popovers are not visible for example.

Would it be possible to lower the z-index of our history portal instead? So that it's between between the default z-index and all the modal/popover/notification ones

Reply:
Great idea! I think that might work

Reply:
I think it's working and probably means the update to the DS was not necessary 🤦‍♂️ but oh well. Good thinking, could you test it out on your end too?